projects/1.20/assets/biomes-o-plenty-delight/biomesoplentydelight/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+  "item.biomesoplentydelight.cooked_bramble": "煮熟的荆条",
+  "item.biomesoplentydelight.barley_bread": "大麦面包",
+  "item.biomesoplentydelight.flesh": "肉块",
+  "item.biomesoplentydelight.glowworm_ice_cream": "萤火虫冰淇淋",
+  "item.biomesoplentydelight.rose_quartz_shard": "玫瑰石英碎片",
+
+  "item.biomesoplentydelight.barley_dough": "大麦面团",
+  "item.biomesoplentydelight.barley_rolls": "大麦卷",
+  "item.biomesoplentydelight.bramble_and_spider_egg": "荆棘和蜘蛛卵",
+  "item.biomesoplentydelight.bramble_pasta": "荆棘意面",
+  "item.biomesoplentydelight.bramble_pasta_with_spider_egg": "荆棘意面配蜘蛛卵",
+  "item.biomesoplentydelight.bramble_sandwich": "荆棘三明治",
+  "item.biomesoplentydelight.cattail_shake": "香蒲奶昔",
+  "item.biomesoplentydelight.cooked_spider_egg": "煎蜘蛛卵",
+  "item.biomesoplentydelight.glowworm_jem": "萤火虫果酱",

```

```suggestion
  "item.biomesoplentydelight.glowworm_jem": "荧光蛛果酱",
```
这些物品的配方是什么样的

如果评论上带有Pending标，可以去Files changed栏下右上角Review changes里选Comment然后Submit review，否则这些评论仅发布者自己可见